requirements.txt

```diff
@@ -0,0 +1,49 @@
+requests
+logzero
+aiohttp-cors
+wheel
+mysqlclient ==1.4.4
+sentry-sdk
+celery == 5.0.5
+azure-storage-blob == 12.8.0
+
+# Django
+# ------------------------------------------------------------------------------
+django-enumchoicefield
+django-dotenv
+django-filter
+django-heroku
```

Comment:
Tror vi kan fjerne denne siden vi går vekk fra heroku eller er det for tidlig? https://pypi.org/project/django-heroku/

Reply:
Prøvde det, men den inneholder noen subdependencies som gir bedre hosting av static sites og tilkoblig til database og litt sånt utils som er kjekt å ha